### DISCUSSION TITLE
Black screen After installing a debloated win 11 pro

### DISCUSSION THREAD

**Discussion by DrMental** [ChrisTitusTech/winutil#1413] [Q&A]
I'm getting a black  screen After installing windows 11 pro on my pc and booting into the main screen then the file explorer open on a black background,  in order to use the desktop I have to restart my pc 

I'm using micro win to debloat my iso 

Help

**Comment by theriverlethe** [comment]
I had the same issue. After a very long time the desktop finally appears but a lot of other things such as task manager don't work normally until rebooting. Afterwards winget wasn't working properly but fortunately there's a handy "winget reinstall" under "config" in winutil. Everything seems to be working now but I would not recommend a "MicroWin" debloated ISO to anyone at this point. 

Edit: Also Win11 Pro (23H2) ISO here.
